imagebuildah/build.go

```diff
@@ -220,7 +220,11 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options BuildOpt
 			// context directory to it.
 			dinfo, err := os.Stat(dfile)
 			if os.IsNotExist(err) {
-				dfile = filepath.Join(options.ContextDirectory, dfile)
+				// if "/workDir" == filepath.Dir("/workDir/Dockerfile")
+				// so don't joint it
+				if options.ContextDirectory != filepath.Dir(dfile) {
```

Comment:
Should this be strings.HasPrefix(dfile, options.ContextDirectory)

Reply:
done